src/rest/Facet/FacetSearchRequest.ts

```diff
@@ -1,5 +1,16 @@
 import { IQuery } from '../Query';
 
+export enum FacetSearchType {
```

Comment:
If we want the enum values to appear in the generated documentation, I believe we'll have to add a comment to it as such:

```
/**
 * The facet types against which facet search is allowed.
 */
```

Reply:
Oh, TIL! Next time I'll make sure to add a comment.

I'm adding your comment.